docs/buildah-bud.md

```diff
@@ -338,9 +346,7 @@ process.
 
 **--platform**="Linux"
```

Comment:
I think the default is wrong as it's in the wrong format. I suggest to keep it empty.